src/check.ts

```diff
@@ -259,6 +259,7 @@ export interface CommonIssue {
    * Only available if the server runs in Targets mode.
    */
   scoring?: GoalScoring;
+  aiRephraseHint: string;
```

Comment:
@dj-delima Also could you write a doc comment on the field, stating from which platform version it is available? It will help external developers to know if they can use it or not